range.go

```diff
@@ -158,6 +164,11 @@ func (p *Plugin) EvaluateThreshold(perfData ...PerformanceData) error {
 
 		if perfData[i].Warn != "" {
 			warningThresholdObject := ParseRangeString(perfData[i].Warn)
+			if warningThresholdObject == nil {
+				err := fmt.Errorf("failed to parse critical range string %s: %w ", perfData[i].Warn, ErrInvalidRangeThreshold)
+				p.ExitStatusCode = StateUNKNOWNExitCode
```

Comment:
Did you mean to write `warning` instead?

```diff
-err := fmt.Errorf("failed to parse critical range string %s: %w ", perfData[i].Warn, ErrInvalidRangeThreshold)
+err := fmt.Errorf("failed to parse warning range string %s: %w ", perfData[i].Warn, ErrInvalidRangeThreshold)
```

Reply:
grrh, copy/past error. fixed in next commit